src/components/designer/taro/actions/MintAssetModal.tsx

```diff
@@ -57,10 +57,7 @@ const MintAssetModal: React.FC<Props> = ({ network }) => {
   useEffect(() => {
     //When polar is first opened, we need to populate the state with the lightning node data
     const lndNode = network.nodes.lightning.find(n => n.name === thisTaroNode?.lndName);
-    if (lndNode) {
-      getWalletBalance(lndNode);
-      getAssets(thisTaroNode);
-    }
+    lndNode && getWalletBalance(lndNode) && getAssets(thisTaroNode);
```

Comment:
This is a clever use of the language but less readable than just wrapping in an if statement. I tend to shy away from syntax like this for maintainability of the project.